### PR TITLE
added fab

### PR DESCRIPTION
**Pull request by regmarcos**
Hi guys! I just added the floating actions buttons to get all the characters from remote and repository and also a button to clear out the recycler.

pr6

### CODE REVIEW

File: data/src/main/java/com/globant/data/database/CharacterDatabase.kt [hunk 1]
```diff
@@ -16,6 +16,18 @@ class CharacterDatabase {
...
+        val mapper = CharacterMapperLocal()
+        var allCharacters = emptyList<MarvelCharacter>()
+        Realm.getDefaultInstance().use {realm ->
+            val result = realm.where(MarvelCharacterRealm::class.java).findAll()
+            result.let {
+                allCharacters = realm.copyFromRealm(it).map { result -> mapper.transform(result) }
+                return Result.Success(allCharacters)
+            }
+        }
+        return Result.Failure(Exception("Not found"))
```
Comment (BQuinteros) [inline data/src/main/java/com/globant/data/database/CharacterDatabase.kt]: pls try to use EMPTY_STRING utils

**Review by SartiMau — APPROVED** [review]

**Review by lucianoeli — APPROVED** [review]

**Review by nicolbidone — APPROVED** [review]

**Review by SartiMau — DISMISSED** [review]

**Review by SartiMau — COMMENTED** [review]
The storage FAB, I see that it only loads the info from the DB, possibly when the service responds, you save it in Realm... but it shouldn't have another icon in the UI and besides have a different name?

**Comment by regmarcos** [comment]
i have changed the icon as you request Mauro
Anotación 2020-02-19 101458

File: app/src/main/java/com/globant/activities/CharactersActivity.kt [hunk 2]
```diff
@@ -9,19 +9,13 @@ import com.globant.adapter.CharacterAdapter
...
 import com.globant.utils.TAG
 import com.globant.viewmodels.RecyclerCharactersViewModel
 import com.globant.viewmodels.RecyclerData
 import com.globant.viewmodels.RecyclerStatus
-import kotlinx.android.synthetic.main.activity_main_recyclerview.fab_delete
-import kotlinx.android.synthetic.main.activity_main_recyclerview.fab_refresh
-import kotlinx.android.synthetic.main.activity_main_recyclerview.fab_storage
-import kotlinx.android.synthetic.main.activity_main_recyclerview.progress_bar_main_activity
-import kotlinx.android.synthetic.main.activity_main_recyclerview.recycler_view_characters
+import kotlinx.android.synthetic.main.activity_main_recyclerview.*
```
Comment (SartiMau) [inline app/src/main/java/com/globant/activities/CharactersActivity.kt]: don't use wildcard imports pls

**Review by SartiMau — CHANGES_REQUESTED** [review]

File: app/src/main/java/com/globant/activities/CharactersActivity.kt (hunk 2 above)
Comment (regmarcos) [reply app/src/main/java/com/globant/activities/CharactersActivity.kt]: done

**Review by SartiMau — APPROVED** [review]

**Pull request merged**